.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Before this pull request can be accepted the following must be true.
+
+- [ ] Unit tests pass (Travis integration).
+- [ ] No code quality issues (Codacy integration).
+- [ ] There are no bugs against the JIRA issues in the current sprint.
```

Comment:
For JIRA stuff it might be worth splitting this into a 'BJSS' use only part as any external pull requesters will not have any access to this